packages/block-library/src/navigation/style.scss

```diff
@@ -535,8 +435,7 @@
 	@include break-small() {
 		&:not(.hidden-by-default) {
 			&:not(.is-menu-open) {
-				display: flex;
-				flex-direction: row;
+				display: contents;
```

Comment:
Is [the Safari/a11y bug](https://caniuse.com/css-display-contents) relevant here?

Reply:
Oooh, good point! I'll look into it.

Reply:
Can confirm `display: contents` wipes out list semantics. Looking at a fix now. Thanks for bringing it up!

Reply:
And here's a fix : #36292

Reply:
No problem, kudos for digging into that and figuring out an alternative 👍🏻